common/djangoapps/third_party_auth/tests/specs/test_lti.py

```diff
@@ -0,0 +1,99 @@
+"""
+Integration tests for third_party_auth LTI auth providers
+"""
+import unittest
+from django.contrib.auth.models import User
+from django.core.urlresolvers import reverse
+from oauthlib.oauth1.rfc5849 import Client, SIGNATURE_TYPE_BODY
+from third_party_auth.tests import testutil
+
+FORM_ENCODED = 'application/x-www-form-urlencoded'
+
+LTI_CONSUMER_KEY = 'consumer'
+LTI_CONSUMER_SECRET = 'secret'
+LTI_TPA_LOGIN_URL = 'http://testserver/auth/login/lti/'
+LTI_TPA_COMPLETE_URL = 'http://testserver/auth/complete/lti/'
+LTI_USER_ID = 'lti_user_id'
+EDX_USER_ID = 'test_user'
+EMAIL = 'lti_user@example.com'
+
+@unittest.skipUnless(testutil.AUTH_FEATURE_ENABLED, 'third_party_auth not enabled')
```

Comment:
I'll look into it but might not be qualified; I copied this approach from third_party_auth/tests/specs/test_testshib.py so maybe @bradenmacdonald has some ideas?